src/pages/test-report/__tests__/test-report.spec.ts

```diff
@@ -27,7 +28,9 @@ describe('TestReportPage', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [
+      declarations: [TestReportPage,
+        MockComponent(CompetencyComponent),
```

Comment:
This one is already in here (2 lines below) and ManoeuvresPopoverComponent should be MockComponent(ManoeuvresPopoverComponent)